react-native-reanimated.d.ts

```diff
@@ -311,13 +311,7 @@ declare module 'react-native-reanimated' {
     ): void
 
     // configuration
-
-    // `addWhitelistedNativeProps` will likely be removed soon, and so is
-    // intentionally not exposed to TypeScript. If it is needed, it could be
-    // uncommented here, or just use
-    // `(Animated as any).addWhitelistedNativeProps({ myProp: true });`
-
-    // addWhitelistedNativeProps(props: { [key: string]: true }): void;
+    export function addWhitelistedNativeProps(props: { [key: string]: true }): void;
```

Comment:
@msand I did not need to use this function in order to make the path animation work on Android. Is it a performance optimization to use it?

Reply:
@wcandillon I think without this the change might go from native to js and back to native, rather than stay native. But, I'm not sure, haven't read the latest source, nor placed any debug breakpoints to test this hypothesis.

Reply:
So yes, animating string includes JS trip, but it's ok, no?

Reply:
Or no, it doesn't. Hmmm, need to check!

Reply:
Sorry, yes it's animating with native drivers as well 😅